main.py

```diff
@@ -1,6 +1,18 @@
 # KaanBOT is a turkish discord bot. It is programmed to work in a single server.
 
 # Importing packages.
+import os
+import requests
+
+try:
+  import requests
```

Comment:
I did not really understand why you imported requests module in two different location. There are at line 4 and line 8.